atom/browser/api/atom_api_web_contents.cc

```diff
@@ -1634,6 +1662,7 @@ void WebContents::BuildPrototype(v8::Isolate* isolate,
                  &WebContents::UnregisterServiceWorker)
       .SetMethod("inspectServiceWorker", &WebContents::InspectServiceWorker)
       .SetMethod("print", &WebContents::Print)
+      .SetMethod("printerList", &WebContents::GetPrinterList)
```

Comment:
Could you add a new test for this method? Even if it just asserts that an array is returned, it will at least verify that no crashes occur on each platform and will give some coverage of this API.